www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -1,13 +1,19 @@
 <?php
+
```

Comment:
tiny nit: we don't need these blank spaces at the top of files for Backdrop (and we are slowly removing them from core). These blank lines are leftovers from the CVS days of Drupal when we used to add things to the top of each file...